tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf

```diff
@@ -76,6 +88,18 @@ Object.Pipeline {
 				copier_type	"SSP"
 				stream_name	"$SPEAKER_CODEC_NAME"
 				node_type	$I2S_LINK_OUTPUT_CLASS
+				IncludeByKey.SSP_SPK_FMT_24 {
+				"true"	{
+						Object.Base.output_audio_format [
+							{
+								out_bit_depth		32
+								out_valid_bit_depth	24
+								out_sample_type		$SAMPLE_TYPE_MSB_INTEGER
+								out_fmt_cfg		"$[($out_channels | ($out_valid_bit_depth * 256))]"
+							}
+						]
+					}
+				}
```

Comment:
so here you don't have a 'false' case ...

Reply:
The default dai input/output format is 32 bits in mixout-gain-eqiir-dts-dai-copier-playback.conf.

mixout-gain-eqiir-dts-dai-copier-playback [
                        Object.Base.audio_format [
                                {
                                        in_bit_depth            32
                                        in_valid_bit_depth      32
                                        out_bit_depth           32
                                        out_valid_bit_depth     32
                                }
                        ]
]
So I don't write the 'false' case. For better reading, I could add 'false' case.